test/unit/Dredd-test.js

```diff
@@ -120,9 +121,9 @@ describe('Dredd class', () => {
         if (error) { return done(error); }
         assert.lengthOf(dredd.files, 3);
         assert.deepEqual(dredd.files, [
-          `${CWD}/test/fixtures/multifile/greeting.apib`,
-          `${CWD}/test/fixtures/multifile/message.apib`,
-          `${CWD}/test/fixtures/multifile/name.apib`,
+          path.join(CWD, '/test/fixtures/multifile/greeting.apib'),
```

Comment:
Since you're using path.join, perhaps we shouldn't hard-code the rest of the path separators in this too?

```js
path.join(CWD, 'test', 'fixtures', 'multifile', 'greeting.apib'),
```

Same applies to all joins below.

Reply:
If it's for the sake of portability, `path.join()` can correctly handle this on Windows.

(The reason why Windows tests were failing was actually the fact I wasn't using `path.join()` everywhere in the tests, which translates the separators.)

If it's for the sake of readability, the current version feels better (to me).